core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/WorkspaceMigrationMonitorSpec.scala

```diff
@@ -0,0 +1,389 @@
+        )
+      }
+
+      // We need a temp bucket to transfer the workspace bucket contents into
+      val migration = runAndWait(
+        WorkspaceMigrationMonitor.workspaceMigrations
+          .filter(_.workspaceId === minimalTestData.v1Workspace.workspaceIdAsUUID)
+          .result
+      )
+        .head
+        .copy(
+          tmpBucketName = GcsBucketName("tmp-bucket-name").some
+        )
+
+      val (job, writeAction) = WorkspaceMigrationMonitor.startStorageTransferJobToTmpBucket(
+        migration,
+        minimalTestData.v1Workspace,
+        GoogleProject("to-be-determined"),
+        mockStsService
+      ).unsafeRunSync
+
+      runAndWait(writeAction) shouldBe()
+
+      val transferJobs = runAndWait(
+        WorkspaceMigrationMonitor.storageTransferJobs
+          .filter(_.migrationId === migration.id)
+          .result
+      )
+
+      transferJobs.length shouldBe 1
+      transferJobs.head.jobName.value shouldBe job.getName
+      transferJobs.head.originBucket.value shouldBe minimalTestData.v1Workspace.bucketName
+      transferJobs.head.destBucket shouldBe migration.tmpBucketName.get
+    }
+  }
+
+  "startStorageTransferJobToFinalBucket" should "create and start a storage transfer job from the tmp bucket to the new workspace bucket" in {
+    withMinimalTestDatabase { _ =>
+      runAndWait {
+        DBIO.seq(
+          workspaceQuery.createOrUpdate(minimalTestData.v1Workspace),
+          WorkspaceMigrationMonitor.schedule(minimalTestData.v1Workspace)
+        )
+      }
+
+      // We need a temp bucket to transfer the workspace bucket contents into
+      val migration = runAndWait(
+        WorkspaceMigrationMonitor.workspaceMigrations
+          .filter(_.workspaceId === minimalTestData.v1Workspace.workspaceIdAsUUID)
+          .result
+      )
+        .head
+        .copy(
+          tmpBucketName = GcsBucketName("tmp-bucket-name").some
+        )
+
+      val (job, writeAction) = WorkspaceMigrationMonitor.startStorageTransferJobToFinalBucket(
+        migration,
+        minimalTestData.v1Workspace,
+        GoogleProject("to-be-determined"),
+        mockStsService
+      ).unsafeRunSync
+
+      runAndWait(writeAction) shouldBe()
+
+      val transferJobs = runAndWait(
+        WorkspaceMigrationMonitor.storageTransferJobs
+          .filter(_.migrationId === migration.id)
+          .result
+      )
+
+      transferJobs.length shouldBe 1
+      transferJobs.head.jobName.value shouldBe job.getName
+      transferJobs.head.originBucket.value shouldBe migration.tmpBucketName.get.value
+      transferJobs.head.destBucket.value shouldBe minimalTestData.v1Workspace.bucketName
+    }
+  }
+
+  "createFinalBucket" should "create a new bucket in the same region as the tmp workspace bucket" ignore {
+    val sourceProject = "general-dev-billing-account"
+    val sourceBucket = "az-leotest"
+    val destProject = "terra-dev-7af423b8"
```

Comment:
Similar to the other change, can we remove these "real" values?  There's another occurrence around line 111 too.

Reply:
we actually use these when we run the test manually so not yet